<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smallChooseForm</class>
 <widget class="QWidget" name="smallChooseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>公告窗口</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabHead">
      <attribute name="title">
       <string>教务公告</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableView" name="tableViewAdmin"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDepart">
      <attribute name="title">
       <string>院系公告</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableView" name="tableViewDepart"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pbtAddAnnounce">
              <property name="text">
               <string>新增公告</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbtSubAnnounce">
              <property name="text">
               <string>删除公告</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateAnnounce">
              <property name="text">
               <string>修改公告</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="labIndex">
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="cbxInterval">
             <property name="text">
              <string>间隔显示</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbxDesOrder">
             <property name="text">
              <string>按序号降序排序</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbxChangColWidth">
             <property name="text">
              <string>调整列宽</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
